pages/api/v3/binder/create.ts

```diff
@@ -0,0 +1,69 @@
+import { NextApiRequest, NextApiResponse } from 'next'
+import { ApiResponse, ListResponse } from '..'
+import middleware from '@pages/api/database/middleware'
+import Cors from 'cors'
+import { POST } from '@constants/http-methods'
+import { cardsQuery } from '@pages/api/database/database'
+import SQL from 'sql-template-strings'
+import { checkUserAuthorization } from '../lib/checkUserAuthorization'
+import { StatusCodes } from 'http-status-codes'
+import methodNotAllowed from '../lib/methodNotAllowed'
+
+const allowedMethods: string[] = [POST] as const
+const cors = Cors({
+  methods: allowedMethods,
+})
+
+export default async function create_binder(
+  req: NextApiRequest,
+  res: NextApiResponse<ApiResponse<ListResponse<''>>>
```

Comment:
This won't cause any problems, but this doesn't need to a `ListResponse` based on what's being returned in the code. Seems like it could just be `res: NextApiResponse<ApiResponse<null>>`